content/references/protocol-reference/transactions/transaction-types/paymentchannelcreate.md

```diff
@@ -10,7 +10,7 @@ labels:
 
 _Added by the [PayChan amendment][]._
 
-Create a unidirectional channel and fund it with XRP. The address sending this transaction becomes the "source address" of the payment channel.
+Create a [payment channel](payment-channels.html) and fund it with XRP. The address sending this transaction becomes the "source address" of the payment channel.
```

Comment:
The word unidirectional, while accurate, is irrelevant and possibly misleading since the XRP Ledger _only_ supports unidirectional payment channels. Also, links the relevant background reading.